dali/operators/reader/video_reader_decoder_cpu_op.h

```diff
@@ -19,7 +19,7 @@
 #include "dali/operators/reader/loader/video/video_loader_decoder_cpu.h"
 
 namespace dali {
-class VideoReaderDecoderCpu : public DataReader<CPUBackend, VideoSample<CPUBackend>> {
+class VideoReaderDecoderCpu : public DataReader<CPUBackend, VideoSample<CPUBackend>, VideoSample<CPUBackend>, true> {
```

Comment:
This applies to all checkpointing PRs:
I recommend adding a template alias:
```C++
template <typename Backend, typename LoadTarget, typename ParseTarget = LoadTarget>
using CheckpointingDataReader = DataReader<Backend, LoadTarget, ParseTarget, true>;
```
to avoid retyping the ParseTarget manually all over the place.